### PR TITLE
Simplify PAGE events

### LINKED ISSUE #156: Simplify PAGE events

### PR DESCRIPTION
**Pull request by gtupak**
Removed referrer and UTM tracking. Referrer is now added automatically to each PAGE event. Also
removed FIRST_PAGE_VISIT events and replaced them by the simpler PAGE events. Refactored the
constant events into an `Events` enum.

BREAKING CHANGE: Removed `trackReferrer` and `trackUTM` config options. Removed parameter inside the
`page()` call.

closes #156

### CODE REVIEW

File: src/constants/events.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+  PAGE = 'PAGE',
+  CONNECT = 'CONNECT',
+  TRANSACTION = 'TRANSACTION_SUBMITTED',
+  ATTRIBUTION = 'ATTRIBUTION',
+  DISCONNECT = 'DISCONNECT',
+  CHAIN_CHANGED = 'CHAIN_CHANGED',
+  TRANSACTION_TRIGGERED = 'TRANSACTION_TRIGGERED',
+  SIGNING_TRIGGERED = 'SIGNING_TRIGGERED',
+  CLICK = 'CLICK',
+}
```
Comment (classicus-eth) [inline src/constants/events.ts:11]: 👍🏻

File: src/constants/config.ts [hunk 2]
```diff
@@ -3,8 +3,6 @@ import { SdkConfig } from '../types'
 export const DEFAULT_SDK_CONFIG: SdkConfig = {
   cacheIdentity: true,
   trackPages: true,
-  trackReferrer: true,
-  trackUTM: true,
```
Comment (classicus-eth) [inline src/constants/config.ts:7]: SDK installers can no longer opt-out of UTM and referrer tracking 🥇 Please correct me only if that's wrong

File: src/ArcxAnalyticsSdk.ts [hunk 3]
```diff
@@ -455,15 +421,16 @@ export class ArcxAnalyticsSdk {
...
-    }
-    return this.event(PAGE_EVENT, attributes)
+  /**
+   * Logs the current page
+   */
+  page(): void {
+    return this.event(Events.PAGE, {
+      referrer: document.referrer,
+    })
   }
```
Comment (classicus-eth) [inline src/ArcxAnalyticsSdk.ts:434]: There any `PAGE` event that may have a `document.referrer` will be added to the `event.referrer` property. 

Ideally there should only be one per `storage_session_id` like we talked about yesterday, but let's see if that assumption holds true once data starts coming in.

@tanghyd This is the one-liner that is responsible for the `referrer` headaches 😆

**Review by classicus-eth — APPROVED** [review]
Love seeing a net-negative lines added PR ☕

File: src/ArcxAnalyticsSdk.ts (hunk 3 above)
Comment (gtupak) [reply src/ArcxAnalyticsSdk.ts:434]: That's right

**Pull request merged**